test/test_generators.py

```diff
@@ -103,28 +284,38 @@ def test_full_generator_windows(tmpdir):
     st_window_shape, min_size=1, max_size=10, unique=True)
 
 
-def create_step_and_image_strategy(limit):
-
+def create_step_and_image_strategy(args):
+    limit, dtype = args
     step_size = st.tuples(
         st.integers(min_value=1, max_value=limit[0]),
         st.integers(min_value=1, max_value=limit[1]),
     )
 
+    if np.issubdtype(dtype, np.floating):
+        elements = st.floats(min_value=-1, max_value=1)
```

Comment:
really -1? not between 0 and 1? scikit image issue?

Reply:
Changed it to 0 and 1 since we normalize between 0 and 1 in other cases.